INSTALL.md

```diff
@@ -1,43 +1,139 @@
 # 1. Back-end setup
 
-Follow the [Sheer installation instructions](https://github.com/cfpb/sheer#installation)
-to get Sheer installed.
-**NOTE:** We suggest creating a virtualenv variable specific to this project,
-such as `cfgov-refresh` instead of `sheer` used in the Sheer installation instructions:
+## Virtualenv & Virtualenvwrapper Python modules
 
+[Install Virtualenv](https://virtualenv.pypa.io/en/latest/index.html) and
+[virtualenvwrapper](https://virtualenvwrapper.readthedocs.org/en/latest/)
+to create a local environment for your server:
 ```bash
-$ mkvirtualenv cfgov-refresh
+pip install virtualenv virtualenvwrapper
 ```
 
-Install the following dependencies into your virtual environment.
-We called ours `cfgov-refresh` (see previous step above):
+## Autoenv module
 
+[Install Autoenv](https://github.com/kennethreitz/autoenv#install) however
+you'd like. (We use [Homebrew]((http://brew.sh)):
+
+Run:
+```bash
+brew install autoenv
+```
+
+After installation, Homebrew will output instructions similar to:
+```bash
+To finish the installation, source activate.sh in your shell:
+  source /Users/[YOUR MAC OSX USERNAME]/homebrew/opt/autoenv/activate.sh
+```
+
+Any time you run the project you'll need to run that last line. If you'll
+be working with the project consistently, we suggest adding it to your bash
+profile by running:
+```bash
+echo 'source /Users/[YOUR MAC OSX USERNAME]/homebrew/opt/autoenv/activate.sh' >> ~/.bash_profile
```

Comment:
Probably need to recommend a different file here for our team, given [our `.bash_profile` setup](https://github.com/cfpb/dotfiles/blob/master/.bash_profile).